src/api/migrations/versions/d4e5f6a7b8c9_add_creator_activated_at.py

```
@@ -0,0 +1,50 @@
+revision = "d4e5f6a7b8c9"
+down_revision = "c6a4f8d9b2e1"
+branch_labels = None
+depends_on = None
+
+
+def _column_exists(table_name: str, column_name: str) -> bool:
+    bind = op.get_bind()
+    inspector = sa.inspect(bind)
+    try:
+        columns = inspector.get_columns(table_name)
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)